app/uk/gov/hmrc/cataloguefrontend/leakdetection/LeakDetectionConnector.scala

```diff
@@ -68,6 +68,14 @@ class LeakDetectionConnector @Inject() (
     )
   }
 
+  def leakDetectionBranchSummaries(repo: String)(implicit hc: HeaderCarrier): Future[Seq[LeakDetectionRepositorySummary]] = {
+    implicit val ldrs: Reads[LeakDetectionRepositorySummary] = LeakDetectionRepositorySummary.reads
+    http.GET[Seq[LeakDetectionRepositorySummary]](
+      url"$url/api/branches/summary?repository=$repo",
+      headers = Seq("Accept" -> "application/json")
```

Comment:
Is the header needed? You'd hope that it could be inferred from the provided `reads` - maybe something http-verbs should help with? guess not easily...